documentation/Installing-Clear-Containers-on-Fedora-25.md

```diff
@@ -0,0 +1,60 @@
+#Install the Clear Containers runtime on Fedora 25
+
+##Install the Clear Containers runtime
+```
+$ sudo -E dnf config-manager --add-repo
+http://download.opensuse.org/repositories/home:clearlinux:preview:clear-containers-2.1/Fedora\_25/home:clearlinux:preview:clear-containers-2.1.repo
+$ sudo dnf install cc-oci-runtime linux-container
+```
+
+##Install docker 1.12
+```
+$ sudo -E dnf config-manager --add-repo https://yum.dockerproject.org/repo/testing/fedora/25
```

Comment:
I simply pulled the installation directions as is which were used in wiki before (and have been used/verified).  Having said that, I can verify and update this.